website/src/common/DisplayList.tsx

```diff
@@ -39,33 +47,24 @@ export default function DisplayList(props) {
         ))}
       </div>
       <div className="w-full overflow-x-auto">
-        <table className="w-full">
-          <thead>
-            <tr>
+        <Table>
+          <TableHeader>
+            <TableRow>
               {Object.keys(data[0]).map((item, key) => (
-                <th
-                  className="border-b min-w-[150px] text-left font-semibold py-2 border-front"
-                  key={key}
-                >
-                  {item}
-                </th>
+                <TableHead key={key}>{item}</TableHead>
               ))}
-            </tr>
-          </thead>
-          <tbody>
-            {data.map((val, key) => {
-              return (
-                <tr key={key} className="hover:bg-accent cursor-default">
-                  {Object.keys(val).map((item, key) => (
-                    <td className="border-b py-3 text-left" key={key}>
-                      {val[item]}
-                    </td>
-                  ))}
-                </tr>
-              );
-            })}
-          </tbody>
-        </table>
+            </TableRow>
+          </TableHeader>
+          <TableBody>
+            {data.map((val, key) => (
+              <TableRow key={key} className="hover:bg-accent cursor-default">
+                {Object.keys(val).map((item, key) => (
+                  <TableCell key={key}>{val[item]}</TableCell>
+                ))}
+              </TableRow>
```

Comment:
I think to go one step further we could also replace the "badge" we are using to show the benchmark's status: finished, failed, started. [We currently use pure CSS ](https://github.com/vitessio/arewefastyet/blob/c36d29a26e5b5a3bf62f97b9292891a4c1b718d3/website/src/pages/StatusPage/components/PreviousExecutions.tsx#L70-L79)but we could use Shadcn too https://ui.shadcn.com/docs/components/badge/